presto-blackhole/src/main/java/io/prestosql/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -62,7 +62,7 @@
     public static final String SCHEMA_NAME = "default";
 
     private final List<String> schemas = new ArrayList<>();
-    private final Map<String, BlackHoleTableHandle> tables = new ConcurrentHashMap<>();
+    private final Map<SchemaTableName, BlackHoleTableHandle> tables = new ConcurrentHashMap<>();
```

Comment:
Is there a check now that disallows table with same name in different schemas?
I don't see it.